content/pages/1.discover/05.stacks/index.en.md

```diff
@@ -0,0 +1,50 @@
+---
+nav_title: Stacks
+---
+
+# Stacks TDP
+
+The following tables show the core components of TDP as well as the Apache branch they were based on, the `-basic` branch which serves as base for our work and the `-fix` branch which includes patches for the correct functionning.
```

Comment:
We shall start by defining what is a stack, eg a collection of components with their associated version named after Ambari and Hortonworks deployment model.